Description: Upstream changes introduced in version 2.2+git20110628-2ubuntu2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 lcms2 (2.2+git20110628-2ubuntu2) oneiric; urgency=low
 .
   * Convert library packages to multiarch. LP: #836004.
 .
 The person named in the Author field signed this changelog entry.
Author: Matthias Klose <doko@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/836004

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- lcms2-2.2+git20110628.orig/libtool
+++ lcms2-2.2+git20110628/libtool
@@ -2,7 +2,7 @@
 
 # libtool - Provide generalized library-building support services.
 # Generated automatically by config.status (lcms2) 2.2
-# Libtool was configured on host till:
+# Libtool was configured on host octogon:
 # NOTE: Changes made to this file will be lost: look at ltmain.sh.
 #
 #   Copyright (C) 1996, 1997, 1998, 1999, 2000, 2001, 2003, 2004, 2005,
@@ -272,10 +272,10 @@ finish_eval=""
 hardcode_into_libs=yes
 
 # Compile-time system search path for libraries.
-sys_lib_search_path_spec="/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1 /lib /usr/lib "
+sys_lib_search_path_spec="/usr/lib/gcc/x86_64-linux-gnu/4.6.1 /usr/lib/x86_64-linux-gnu /usr/lib /lib/x86_64-linux-gnu /lib "
 
 # Run-time system search path for libraries.
-sys_lib_dlsearch_path_spec="/lib /usr/lib /lib32 /usr/lib32 /usr/local/lib /lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu/mesa-egl /usr/lib/x86_64-linux-gnu/mesa "
+sys_lib_dlsearch_path_spec="/lib /usr/lib /lib32 /usr/lib32 /usr/local/lib /lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu/mesa "
 
 # Whether dlopen is supported.
 dlopen_support=unknown
@@ -10264,17 +10264,17 @@ file_list_spec=""
 hardcode_action=immediate
 
 # The directories searched by this compiler when creating a shared library.
-compiler_lib_search_dirs="/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1 /usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/../../.."
+compiler_lib_search_dirs="/usr/lib/gcc/x86_64-linux-gnu/4.6.1 /usr/lib/gcc/x86_64-linux-gnu/4.6.1/../../../x86_64-linux-gnu /usr/lib/gcc/x86_64-linux-gnu/4.6.1/../../../../lib /lib/x86_64-linux-gnu /lib/../lib /usr/lib/x86_64-linux-gnu /usr/lib/../lib /usr/lib/gcc/x86_64-linux-gnu/4.6.1/../../.."
 
 # Dependencies to place before and after the objects being linked to
 # create a shared library.
-predep_objects="/usr/lib/x86_64-linux-gnu/crti.o /usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/crtbeginS.o"
-postdep_objects="/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/crtendS.o /usr/lib/x86_64-linux-gnu/crtn.o"
+predep_objects="/usr/lib/gcc/x86_64-linux-gnu/4.6.1/../../../x86_64-linux-gnu/crti.o /usr/lib/gcc/x86_64-linux-gnu/4.6.1/crtbeginS.o"
+postdep_objects="/usr/lib/gcc/x86_64-linux-gnu/4.6.1/crtendS.o /usr/lib/gcc/x86_64-linux-gnu/4.6.1/../../../x86_64-linux-gnu/crtn.o"
 predeps=""
 postdeps="-lstdc++ -lm -lgcc_s -lc -lgcc_s"
 
 # The library search path used internally by the compiler when linking
 # a shared library.
-compiler_lib_search_path="-L/lib/x86_64-linux-gnu -L/usr/lib/x86_64-linux-gnu -L/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1 -L/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/../../.."
+compiler_lib_search_path="-L/usr/lib/gcc/x86_64-linux-gnu/4.6.1 -L/usr/lib/gcc/x86_64-linux-gnu/4.6.1/../../../x86_64-linux-gnu -L/usr/lib/gcc/x86_64-linux-gnu/4.6.1/../../../../lib -L/lib/x86_64-linux-gnu -L/lib/../lib -L/usr/lib/x86_64-linux-gnu -L/usr/lib/../lib -L/usr/lib/gcc/x86_64-linux-gnu/4.6.1/../../.."
 
 # ### END LIBTOOL TAG CONFIG: CXX
--- lcms2-2.2+git20110628.orig/Makefile
+++ lcms2-2.2+git20110628/Makefile
@@ -128,13 +128,13 @@ DIST_ARCHIVES = $(distdir).tar.gz $(dist
 GZIP_ENV = --best
 distuninstallcheck_listfiles = find . -type f -print
 distcleancheck_listfiles = find . -type f -print
-ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -182,7 +182,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -206,10 +206,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
-abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
-abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
-abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628
+abs_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628
+abs_top_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628
+abs_top_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -240,7 +240,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
--- lcms2-2.2+git20110628.orig/src/Makefile
+++ lcms2-2.2+git20110628/src/Makefile
@@ -99,13 +99,13 @@ DIST_SOURCES = $(liblcms2_la_SOURCES)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -153,7 +153,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -177,10 +177,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/src
-abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/src
-abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
-abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628/src
+abs_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628/src
+abs_top_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628
+abs_top_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -213,7 +213,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
--- lcms2-2.2+git20110628.orig/testbed/Makefile
+++ lcms2-2.2+git20110628/testbed/Makefile
@@ -71,13 +71,13 @@ DIST_SOURCES = $(testcms_SOURCES)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -125,7 +125,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -149,10 +149,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/testbed
-abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/testbed
-abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
-abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628/testbed
+abs_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628/testbed
+abs_top_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628
+abs_top_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -183,7 +183,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
--- lcms2-2.2+git20110628.orig/utils/linkicc/Makefile
+++ lcms2-2.2+git20110628/utils/linkicc/Makefile
@@ -76,13 +76,13 @@ DIST_SOURCES = $(linkicc_SOURCES)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -130,7 +130,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -154,10 +154,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/linkicc
-abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/linkicc
-abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
-abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628/utils/linkicc
+abs_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628/utils/linkicc
+abs_top_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628
+abs_top_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -188,7 +188,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
--- lcms2-2.2+git20110628.orig/utils/transicc/Makefile
+++ lcms2-2.2+git20110628/utils/transicc/Makefile
@@ -76,13 +76,13 @@ DIST_SOURCES = $(transicc_SOURCES)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -130,7 +130,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -154,10 +154,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/transicc
-abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/transicc
-abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
-abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628/utils/transicc
+abs_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628/utils/transicc
+abs_top_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628
+abs_top_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -188,7 +188,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
--- lcms2-2.2+git20110628.orig/utils/jpgicc/Makefile
+++ lcms2-2.2+git20110628/utils/jpgicc/Makefile
@@ -100,13 +100,13 @@ MANS = $(man_MANS)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -154,7 +154,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -178,10 +178,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/jpgicc
-abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/jpgicc
-abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
-abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628/utils/jpgicc
+abs_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628/utils/jpgicc
+abs_top_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628
+abs_top_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -212,7 +212,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
--- lcms2-2.2+git20110628.orig/utils/tificc/Makefile
+++ lcms2-2.2+git20110628/utils/tificc/Makefile
@@ -99,13 +99,13 @@ MANS = $(man_MANS)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -153,7 +153,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -177,10 +177,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/tificc
-abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/tificc
-abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
-abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628/utils/tificc
+abs_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628/utils/tificc
+abs_top_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628
+abs_top_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -211,7 +211,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
--- lcms2-2.2+git20110628.orig/utils/psicc/Makefile
+++ lcms2-2.2+git20110628/utils/psicc/Makefile
@@ -75,13 +75,13 @@ DIST_SOURCES = $(psicc_SOURCES)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -129,7 +129,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -153,10 +153,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/psicc
-abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/psicc
-abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
-abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628/utils/psicc
+abs_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628/utils/psicc
+abs_top_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628
+abs_top_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -187,7 +187,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
--- lcms2-2.2+git20110628.orig/include/Makefile
+++ lcms2-2.2+git20110628/include/Makefile
@@ -76,13 +76,13 @@ HEADERS = $(include_HEADERS)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -130,7 +130,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -154,10 +154,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/include
-abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/include
-abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
-abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628/include
+abs_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628/include
+abs_top_builddir = /home/packages/tmp/u/lcms2-2.2+git20110628
+abs_top_srcdir = /home/packages/tmp/u/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -188,7 +188,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/packages/tmp/u/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
